.vscode/settings.json

```diff
@@ -3,5 +3,10 @@
     "launch": {
         "configurations": [],
         "compounds": []
-    }
+    },
+    "C_Cpp.default.configurationProvider": "ms-vscode.cmake-tools",
+    "cmake.generator": "Unix Makefiles",
+    "cmake.preferredGenerators": [
+        "Unix Makefiles"
+    ]
```

Comment:
Make vscode be consistent with our (and the default) cmake makefile generation mechanism.
Also probably doesn't belong here.